@re-/assert/test/metaTests/testHelpers.ts

```diff
@@ -0,0 +1,110 @@
+import { strict } from "node:assert"
+import { fromHere, readJson, shell } from "@re-/node"
+import {
+    ExpressionStatement,
+    Project,
+    SourceFile,
+    SyntaxKind,
+    VariableStatement
+} from "ts-morph"
+
+const EXPECTED_RESULTS_JSON_PATH = fromHere("expectedResults.json")
+const PATH_TO_ASSERTIONS_DIR = fromHere(".reassert")
+
+const getExpressionStatements = (
+    functionName: string,
+    varStatements: VariableStatement[]
+) =>
+    varStatements
+        .find((statement) => statement.getText().includes(functionName))
+        ?.getDescendantsOfKind(SyntaxKind.ExpressionStatement)
+
+const getTestComment = (statement: ExpressionStatement, text: string) => {
```

Comment:
I think `text` in this case refers to the contents of the whole file? I might rename it to `fileText` to make that more immediately clear.